incubation-process.md

```diff
@@ -16,6 +16,8 @@ Projects which intend to join the OpenRail Association are considered candidates
 
 ## Stage 1: Sandbox
 
+![Badge Stage 1](https://openrailassociation.org/badges/openrail-project-stage-1.svg)
```

Comment:
How about linking the badges to their respective chapters in the incubation process? Ideally, this anchor would be static in case the headline changes. I think you can also set that in GitHub's Markdown.

Reply:
I like the idea.

Reply:
Fine with me.